src/_gettsim/time_conversion.py

```diff
@@ -230,8 +231,8 @@ def d_to_w(value: float) -> float:
 
 
 def create_time_conversion_functions(
-    functions_tree: NestedFunctionDict,
-    data_tree: NestedDataDict,
+    functions: NestedFunctionDict,
```

Comment:
```suggestion
    functions: QualNameFunctionDict,
```

etc.

Time to turn mypy on, I think :smile:.